gno.land/pkg/integration/testing_integration.go

```diff
@@ -270,7 +276,54 @@ func setupGnolandTestScript(t *testing.T, txtarDir string) testscript.Params {
 					ts.Fatalf("error creating account %s: %s", args[0], err)
 				}
 
-				newUserBalances = append(newUserBalances, balance)
+				// Add balance to genesis
+				genesis := ts.Value(envKeyGenesis).(*gnoland.GnoGenesisState)
+				genesis.Balances = append(genesis.Balances, balance)
+			},
+			// use load a specific package from the example folder or from the working directory
+			"use": func(ts *testscript.TestScript, neg bool, args []string) {
+				// special dirs
+				workDir := ts.Getenv("WORK")
+				examplesDir := filepath.Join(gnoRootDir, "examples")
+
+				pkgs := ts.Value(envKeyPkgsLoader).(pkgsLoader)
+
+				var path, name string
+				switch len(args) {
+				case 2:
+					name = args[0]
+					path = filepath.Clean(args[1])
+				case 1:
+					path = filepath.Clean(args[0])
+				case 0:
+					ts.Fatalf("`use`: no arguments specified")
+				default:
+					ts.Fatalf("`use`: too many arguments specified")
+				}
+
+				// if `all` is specified fully load example folder
+				// NOTE: in 99% of cases, this is not needed and
+				// packages should be load individually
+				if path == "all" {
+					ts.Logf("warning: loading all packages")
+					err := pkgs.loadPackagesFromDir(examplesDir)
+					if err != nil {
+						ts.Fatalf("unable to load packages from %q: %s", examplesDir, err)
+					}
```

Comment:
Please feel free to resolve even if you don't apply the suggestion.
```suggestion
					if err := pkgs.loadPackagesFromDir(examplesDir); err != nil {
						ts.Fatalf("unable to load packages from %q: %s", examplesDir, err)
					}
```